### PR TITLE
Added ROI support on Ocean. Fix small bug.

### PR DESCRIPTION
**Pull request by gabrielfedel**
Added ROIs (regions of interest) on Ocean. 
Fix a small bug with pv variable.

### CODE REVIEW

File: py4syn/epics/ImageHDFClass.py [hunk 1]
```diff
@@ -46,13 +46,13 @@ def nameFile(self, output, prefix, sufix):
 
         return resultName
 
-    def saveSpectrum(self, snake = True):
+    def saveSpectrum(self, snake = True, sufixName = ""):
```
Comment (hhslepicka) [inline py4syn/epics/ImageHDFClass.py]: Typo at "suffix"

File: py4syn/epics/OceanClass.py [hunk 2]
```diff
@@ -12,8 +16,9 @@
     .. note:: 10/18/2016 [gabrielfedel]  first version released
 """
 import os
+from bisect import bisect
 
-from epics import PV
+from epics import PV, caget
```
Comment (hhslepicka) [inline py4syn/epics/OceanClass.py]: Why are you using caget and not a PV?

**Review by hhslepicka — CHANGES_REQUESTED** [review]
Hi @gabrielfedel, I highlighted some points. Please submit the changes and I will merge it.

File: py4syn/epics/OceanClass.py (hunk 2 above)
Comment (gabrielfedel) [reply py4syn/epics/OceanClass.py]: ops, fixed!

File: py4syn/epics/ImageHDFClass.py (hunk 1 above)
Comment (gabrielfedel) [reply py4syn/epics/ImageHDFClass.py]: fixed!

**Comment by gabrielfedel** [comment]
I fixed this points, removed unnecessary imports and check OceanClass on PEP 8.

**Review by hhslepicka — APPROVED** [review]

**Comment by hhslepicka** [comment]
Great! Thanks @gabrielfedel!

**Pull request merged**